frontend/src/components/common/auto-complete/auto-complete.tsx

```diff
@@ -1,6 +1,7 @@
 import { useState } from 'hooks/hooks';
-import { ClearButton, Option, Typeahead } from 'react-bootstrap-typeahead';
+import { Option, Typeahead } from 'react-bootstrap-typeahead';
```

Comment:
here you need to return the file to its original state,
when you change this button, the component is rendered incorrectly